app/grandchallenge/cases/templates/cases/flexible_image_widget.html

```diff
@@ -5,11 +5,11 @@
         id="widgetSelect-{{ widget.name }}"
         hx-get="{% url 'cases:select-image-widget' %}"
         hx-target="#area-{{ widget.name }}"
-        hx-trigger="widgetSelected, change"
+        hx-trigger="widgetSelected, change, load"
         hx-include="[id='values-{{ widget.name }}']"
 >
     {% if widget.attrs.current_value %}
-        <option value="{{ widget.attrs.current_value.image.pk }}">{{ widget.attrs.current_value }}</option>
+        <option value="{{ widget.attrs.current_value.pk }}">{{ widget.attrs.current_value.title }}</option>

```

Comment:
For my understanding: we don't actually use the `value` of this anywhere, do we? Could it then just be left empty or set to one of the two widget choice types below (depending on the type)?

Reply:
I made the change in [Add WidgetChoice for selected image for semantic consistency](https://github.com/comic/grand-challenge.org/pull/3681/commits/833c47a80a86a529aa15b0f9d6f0981670d6e7fe)
It is not much change but semantically more sense instead of printing the pk value without using it anywhere.